Module-3-Javascript-Coding-Task/src/content-weight.spec.js

```diff
@@ -0,0 +1,91 @@
+import { contentWeight, contentWeightCalc } from './content-weight'
+
+describe('content-weight', () => {
+
+    describe('contentWeight', () => {
+
+        it('should return 80', () => {
+
+            const bottleWeight = 120
+            const scale = '2 times larger'
+            const actual = contentWeight(bottleWeight, scale)
+            const expected = 80
+
+            expect(actual).toEqual(expected)
+        })
+
+        it('should return 40', () => {
+
+            const bottleWeight = 120
+            const scale = '2 times smaller'
+            const actual = contentWeight(bottleWeight, scale)
+            const expected = 40
+
+            expect(actual).toEqual(expected)
+        })
+
+        it('should return empty string when undefined', () => {
+
+            const bottleWeight = undefined
+            const scale = undefined
```

Comment:
`scale` changes are not tested here, since the code has already returned before checking it due to `bottleWeight` being undefined or null

Reply:
corrected to 

         it('should return empty string when undefined', () => {

            const bottleWeight = undefined
            const scale = '2 times smaller'
            const actual = contentWeightCalc(bottleWeight, scale)
            const expected = null

            expect(actual).toEqual(expected)
        })

        it('should return empty string when undefined', () => {

            const bottleWeight = 120
            const scale = undefined
            const actual = contentWeightCalc(bottleWeight, scale)
            const expected = null

            expect(actual).toEqual(expected)
        })

        it('should return empty string when null', () => {

            const bottleWeight = 120
            const scale = null
            const actual = contentWeightCalc(bottleWeight, scale)
            const expected = null

            expect(actual).toEqual(expected)
        })

        it('should return empty string when null', () => {

            const bottleWeight = null
            const scale = '2 times smaller'
            const actual = contentWeightCalc(bottleWeight, scale)
            const expected = null

            expect(actual).toEqual(expected)
        })`